internal/attestation/measurements/measurements.go

```diff
@@ -112,6 +112,35 @@ func (m *M) FetchAndVerify(
 	return hex.EncodeToString(shaHash[:]), nil
 }
 
+// Fetch fetches measurements via provided URLs, using client for download.
+func (m *M) Fetch(ctx context.Context, client *http.Client, measurementsURL *url.URL, metadata WithMetadata) error {
```

Comment:
Not a fan of having this function exported since it is used in tests only, and shares all of its code with `FetchAndVerify`.
Is this function really necessary, or could  it be removed from the `measurements` package and replaced with a simple download of the measurements and unmarshal call in the e2e test?

Reply:
Done.